safe_transaction_service/history/views.py

```diff
@@ -854,6 +913,29 @@ def add_tokens_to_transfers(self, transfers: TransferDict) -> TransferDict:
             transfer["token"] = tokens.get(transfer["token_address"])
         return transfers
 
+    def get_token_transfer(self, transaction_hash: str, log_index: str):
```

Comment:
`get_token_transfer` -> `get_unique_token_transfer` or something like that, and please add some documentation to explain why this method is needed 😉